client/flutter/lib/pages/main_pages/routes.dart

```diff
@@ -32,6 +33,7 @@ class Routes {
           dataSource: FactContent.getTheFacts,
           title: S.of(context).homePagePageButtonWHOMythBusters,
         ),
+    '/recent-numbers': (context) => RecentNumbersPage(),
```

Comment:
No action needed: We should check with David whether pushing this as opposed to selecting the tab is the right way to go for pages that also have tabs.  IMO Pushing is right, as you have it, just for simplicity.